packages/dapp/package.json

```diff
@@ -53,6 +54,7 @@
     "@vercel/speed-insights": "^1.0.10",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
+    "crypto-js": "^4.2.0",
```

Comment:
We probably don't need this package. If we need the `crypto` package we can use  [SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto) which is available in the browser.